ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+
+"""
+ndmg.stats.qa_skullstrip
+~~~~~~~~~~~~~~~~~~~~~~
+The top level ndmg quality assurance for skull strip module.
+In this module, ndmg:
+1. Input original t1w file and the skull-striped brain file
+2. Shows the skull-stripped brain (green) overlaid on the original t1w (magenta)
+3. save the image into output directory
+"""
+
+import warnings
+
+warnings.simplefilter("ignore")
+import os
+import re
+import sys
+import numpy as np
+import nibabel as nb
+from argparse import ArgumentParser
+from scipy import ndimage
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib as mpl
+
+mpl.use("Agg")  # very important above pyplot import
+from nilearn.plotting.edge_detect import _edge_map as edge_map
+import matplotlib.pyplot as plt
+from ndmg.utils.qa_utils import get_min_max, opaque_colorscale, pad_im
+from ndmg.utils.gen_utils import get_filename, get_braindata
+
+
+
+
+
+def gen_overlay_pngs(
+    brain, original, outdir, loc=0, mean=False, minthr=2, maxthr=95, edge=False):
+    """Generate a QA image for skullstrip.
+    will call the function plot_overlays_skullstrip
+
+    Parameters
+    ----------
+    brain: nifti file
+        Path to the skull-stripped nifti brain
+    original: nifti file
+        Path to the original t1w brain, with the skull included
+    outdir: str
+        Path to the directory where QA will be saved
+    loc: int
+    mean: bool
+    minthr: int
+    maxthr: int
+    edge: bool
```

Comment:
You need to define what these variables mean

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/19a993b9f5c2c92ce68c5e335bf0dc384f69651a

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/19a993b9f5c2c92ce68c5e335bf0dc384f69651a